FMB/FiniteModelBuilder.cpp

```diff
@@ -2172,6 +2172,7 @@ void FiniteModelBuilder::onModelFound()
   //}
 
   //Evaluate removed propositions and predicates
+#if 0
```

Comment:
Remember to debug this properly: do we have an issue for this?

Reply:
We have two:
https://github.com/vprover/vampire/issues/461
https://github.com/vprover/vampire/issues/565